dlt/destinations/impl/athena/athena.py

```diff
@@ -178,6 +178,9 @@ def __init__(
         capabilities: DestinationCapabilitiesContext,
     ) -> None:
         super().__init__(None, dataset_name, staging_dataset_name, capabilities)
+        self.INFO_TABLES_QUERY_THRESHOLD = 80
```

Comment:
this is hacky. please add `info_tables_query_threshold` to `AthenaClientConfiguration` and just use it directly. remove `INFO_TABLES_QUERY_THRESHOLD` from the whole codebase